### ISSUE TITLE
Hard to follow

### ISSUE THREAD

**Issue by Cogur** [MicrosoftDocs/OfficeDocs-SkypeForBusiness#4304]
This page uses 3 totally different ways to describe what a guest user can or cannot do.
It would be great if you could have a unified table of what a guest can do.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 048930a4-fdf2-9a90-3b47-46af66db3db4
* Version Independent ID: 8d6a2c84-94e9-b17e-f5c1-91191187ce4e
* Content: [What the guest experience is like - Microsoft Teams](https://docs.microsoft.com/en-us/microsoftteams/guest-experience#feedback)
* Content Source: [Teams/guest-experience.md](https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/blob/live/Teams/guest-experience.md)
* Service: **msteams**
* GitHub Login: @LanaChin
* Microsoft Alias: **v-lanac**

**Comment by scanum** [comment]
@Cogur Thank you for submitting feedback and contributing to the docs. We are currently investigating this.

**Comment by dariomws** [comment]
Hi @Cogur, thank you for your feedback.

There are 2 tables in the article and I think they are hard to merge. What are you suggesting?

**Comment by Cogur** [comment]
Don't know what Teams user in the organization
Meas but i gies its in e1 to e3 or e5 groups. So in that way these two
tables could be merged that way. And you can also merge the faq section in
this table as well i think.

Also as this page is about the guest experience it's not relevant if the
not guest useris a e1 or e3 user. So it should bot block the merge of these
3 sources. I think this Ex stuff could be mentiond as a starred reference
on the bottom of the page like "if you interested in the difference of the
organization user capabilities please check here(link)"

**Comment by dariomws** [comment]
E1, E3 and E5 include Teams license. So yes, they are Teams users.

E5 includes Phone System license, this is why there is another table for calling. In this table, you can see the difference between guest, E1/3 and E5 users.

**Comment by andrePKI** [comment]
Checkmarks versus Yes/No perhaps?

**Comment by dariomws** [comment]
@andrePKI are you suggesting to replace Yes/No with checkmarks in the second table?

**Comment by andrePKI** [comment]
@dariomws Maybe. Iam not sure what the OP was getting at. But IMHO I think that have either checkmarks _or_ yes/no (but not both) is one way to get "a unified table"

**Comment by Cogur** [comment]
My opinion: no one gives a damn thing about the difference between non
guest users on a page ment to tell you what the guesn users can or cannot
do.

I think you should put the two table and the faq par in one big table, and
maibe make an other one to list the difference between none guest users, or
link the page which has more details about these differences.

But this page should be kept stupid (kiss)

**Comment by dariomws** [comment]
Thanks, everyone. I discussed with my partners and we do not agree to merge the tables and faq section in only one table, sorry about that.

But I agree that checkmarks instead of yes/no in the second table could be better. If you agree, I can update the second table with checkmarks.

Thanks!

**Comment by scanum** [comment]
@Cogur We appreciate any feedback that improves the content in the Microsoft docs but we haven't had any response from you so we are going to close this issue.  
 
Please feel free to re-open this issue if there is a specific area of the docs that we can improve or make better. Thank you.
